homeassistant/components/xiaomi_miio/air_quality.py

```diff
@@ -133,3 +165,40 @@ def device_state_attributes(self):
     def unit_of_measurement(self):
         """Return the unit of measurement."""
         return self._unit_of_measurement
+
+
+class AirMonitorS1(AirMonitorB1):
+    """Air Quality class for Xiaomi cgllc.airmonitor.s1 device."""
+
+    async def async_update(self):
+        """Fetch state from the miio device."""
+        try:
+            state = await self.hass.async_add_executor_job(self._device.status)
+            _LOGGER.debug("Got new state: %s", state)
+            self._carbon_dioxide = state.co2
+            self._particulate_matter_2_5 = state.pm25
+            self._total_volatile_organic_compounds = state.tvoc
+            self._available = True
+        except DeviceException as ex:
+            self._available = False
+            _LOGGER.error("Got exception while fetching the state: %s", ex)
+
+
+class AirMonitorV1(AirMonitorB1):
+    """Air Quality class for Xiaomi cgllc.airmonitor.s1 device."""
+
+    async def async_update(self):
+        """Fetch state from the miio device."""
+        try:
+            state = await self.hass.async_add_executor_job(self._device.status)
+            _LOGGER.debug("Got new state: %s", state)
+            self._air_quality_index = state.aqi
+            self._available = True
+        except DeviceException as ex:
+            self._available = False
+            _LOGGER.error("Got exception while fetching the state: %s", ex)
+
+    @property
+    def unit_of_measurement(self):
+        """Return the unit of measurement."""
+        return "AQI"
```

Comment:
This should `return None` since the pm25 isn't reported.

Reply:
done.